apps/gro/fields/name.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   'name-text': {
+    mixin: 'input-text',
     validate: ['required'],
     label: 'fields.name-text.label'
```

Comment:
I thought this had all been done by Geoff - it may be cos I renamed the file it was lost in the merge